### ISSUE TITLE
Code format breaks code

### ISSUE THREAD

**Issue by petr-ujezdsky** [ant-druha/intellij-powershell#36]
```
If (-NOT $global:balloon)
```

is reformatted into

```
If (-NOT$global:balloon)
```

which is unrunnable.
I have stumbled over this when I tried to integrate https://github.com/proxb/PowerShell_Scripts/blob/master/Invoke-BalloonTip.ps1 into our project.

**Comment by pietrygamat** [comment]
This is really critical issue for me too.   
This
```powershell
Start-Job -ScriptBlock { runThis.exe --color=blue --name=jacob }
```
is reformatted to
```powershell
Start-Job -ScriptBlock { runThis.exe --color = blue --name = jacob }
```

Autoformatting applied on git commit makes this a particulary nasty bug.

**Comment by ForNeVeR** [comment]
Closed as a duplicate of #102.
